livekit-plugins/livekit-plugins-openai/livekit/plugins/openai/_oai_api.py

```diff
@@ -115,6 +116,16 @@ def type2str(t: type) -> str:
             if arg_info.choices:
                 p["enum"] = arg_info.choices
 
+            if arg_info.type is int and arg_info.choices and model is not None:
```

Comment:
model is not None doesn't make a ton of sense here? U could just check if it is Vertex or not